shank-render/src/pda/pda_common.rs

```diff
@@ -0,0 +1,44 @@
+use shank_macro_impl::{
+    parsed_struct::{ProcessedSeed, SeedArg, StructAttr, StructAttrs},
+    syn::Result as ParseResult,
+};
+
+pub fn try_process_seeds(
+    struct_attrs: &StructAttrs,
+) -> ParseResult<Vec<ProcessedSeed>> {
+    let all_seeds = struct_attrs
+        .items_ref()
+        .iter()
+        .map(|attr| match attr {
+            StructAttr::Seeds(seeds) => seeds,
+        })
+        .collect::<Vec<_>>();
+
+    assert!(
+        all_seeds.len() <= 1,
+        "Should only have one seed definition per account"
+    );
+
+    if all_seeds.is_empty() {
+        Ok(vec![])
+    } else {
+        let seeds = all_seeds.first().unwrap();
+        seeds.process()
+    }
```

Comment:
Logically fine, but might be better to use pattern matching (Probably with an if let) instead of unwrapping.